lib/runtime-core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ libc = "0.2.60"
 hex = "0.3"
 smallvec = "0.6"
 bincode = "1.1"
+maplit = "1.0.2"
```

Comment:
We should just depend on version "1", because this is used as a library, we want to use whatever version is in the application's lockfile unless we need specific bug fixes to work correctly!

Reply:
The justification is that:
1. duplicated dependencies can cause issues in some cases, for example if semantic versioning is violated and a bug is introduced
2. duplicated dependencies can cause increased binary size and slower compile times